test/runtests.jl

```diff
@@ -75,7 +75,7 @@ POMDPs.transition(::TerminalPOMDP, s, a) = Deterministic(s + a)
     pomdp = TerminalPOMDP()
     pf = BootstrapFilter(pomdp, 100)
     bp = update(pf, initialize_belief(pf, Categorical([0.5, 0.5])), -1, 1.0)
-    @test all(particles(bp) .== 1)
+    @test abs(mean(bp) - 1.0) < 1e-5
```

Comment:
Wouldn't it be better to test that all particles with nonzero weight are 1? or `pdf(bp, 1.0)` is approximately 1?

Reply:
> Wouldn't it be better to test that all particles with nonzero weight are 1

I agree with this. Addressed this in the newer commit